src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/CellTableViewCell.cs

```diff
@@ -72,6 +72,13 @@ internal static UITableViewCell GetPlatformCell(UITableView tableView, Cell cell
 			else
 				reusableCell = tableView.DequeueReusableCell(id);
 
+			if (reusableCell is not null && reusableCell.Hidden)
+			{
+				// set hidden to false doesn't work
+				// force recreate of the whole cell instead
+				reusableCell = null;
```

Comment:
won't this break all cell reuse?

Reply:
No, because those cell return that are not hidden are perfectly fine to reuse. Only those hidden are not. Also, I point out that setting hidden to false doesn't fix it. This may not be a perfect fix, but currently this fix at least make the ListView work correctly.